bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -353,14 +353,17 @@ spec:
             spec:
               affinity:
                 nodeAffinity:
-                  requiredDuringSchedulingIgnoredDuringExecution:
-                    nodeSelectorTerms:
-                    - matchExpressions:
+                  preferredDuringSchedulingIgnoredDuringExecution:
+                  - preference:
+                      matchExpressions:
                       - key: node-role.kubernetes.io/master
                         operator: Exists
-                    - matchExpressions:
+                    weight: 1
```

Comment:
question: is `weight` used to prioritize several  preferences ? (I assume the highest the weight means the preference is in higher prioritization)

Reply:
>If there are two possible nodes that match the preferredDuringSchedulingIgnoredDuringExecution rule, one with the label-1:key-1 label and another with the label-2:key-2 label, the scheduler considers the weight of each node and adds the weight to the other scores for that node, and schedules the Pod onto the node with the highest final score.

From https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity-weight

Reply:
The way I understand it, the weight is added to an `overall` node score (didn't check how it's calculated), so it would be more probably (but not definitive) for a pod to be scheduled on higher weight node.